services/app-api/src/resolvers/contract/updateDraftContractRates.test.ts

```diff
@@ -0,0 +1,686 @@
+                                amendmentEffectiveDateEnd: '2025-02-01',
+                                rateProgramIDs: ['foo'],
+
+                                rateDocuments: [
+                                    {
+                                        s3URL: 'foo://bar',
+                                        name: 'updatedratedoc1.doc',
+                                        sha256: 'foobar',
+                                    },
+                                ],
+                                supportingDocuments: [
+                                    {
+                                        s3URL: 'foo://bar1',
+                                        name: 'ratesupdoc1.doc',
+                                        sha256: 'foobar1',
+                                    },
+                                    {
+                                        s3URL: 'foo://bar2',
+                                        name: 'ratesupdoc2.doc',
+                                        sha256: 'foobar2',
+                                    },
+                                ],
+                                certifyingActuaryContacts: [
+                                    {
+                                        name: 'Foo Person',
+                                        titleRole: 'Bar Job',
+                                        email: 'foo@example.com',
+                                        actuarialFirm: 'GUIDEHOUSE',
+                                    },
+                                ],
+                                addtlActuaryContacts: [
+                                    {
+                                        name: 'Bar Person',
+                                        titleRole: 'Baz Job',
+                                        email: 'bar@example.com',
+                                        actuarialFirm: 'OTHER',
+                                        actuarialFirmOther: 'Some Firm',
+                                    },
+                                ],
+                                actuaryCommunicationPreference:
+                                    'OACT_TO_ACTUARY',
+                                packagesWithSharedRateCerts: [],
+                            },
+                        },
+                    ],
+                },
+            },
+        })
+        expect(result.errors).toBeUndefined()
+
+        if (!result.data) {
+            throw new Error('no data returned')
+        }
+
+        const draftRates =
+            result.data.updateDraftContractRates.contract.draftRates
+
+        const draftFormData = draftRates[0].draftRevision.formData
+
+        expect(draftFormData.rateType).toBe('AMENDMENT')
+        expect(draftFormData.rateDocuments).toHaveLength(1)
+        expect(draftFormData.rateDocuments[0].name).toBe('updatedratedoc1.doc')
+    })
+
+    it('doesnt allow updating a non-linked rate', async () => {
```

Comment:
```suggestion
    it('doesnt allow updating a non-child rate', async () => {
```
I think that is what we are testing here... right?